sentry-logback/src/test/kotlin/io/sentry/logback/SentryAppenderTest.kt

```diff
@@ -213,4 +225,29 @@ class SentryAppenderTest {
             })
         }
     }
+
+    @Test
+    fun `attaches breadcrumbs`() {
```

Comment:
```suggestion
    fun `attaches breadcrumbs with level higher than minimumBreadcrumbLevel`() {
```

Can we have some more use cases here? Like level higher than breadcrumb is captured as event but not added as breadcrumbs.
Set only level for breadcrumb, only for event, etc?